etc/conf.templates/distributed/20-warpscript.conf.template

```diff
@@ -222,3 +222,10 @@ warpscript.mobius.pool = 16
 // allow C style block comments. /**** valid block comment ****/
 //
 warpscript.comments.loose = true
+
+//
+// Default priority order for matching labels when doing a FIND/FETCH.
+// Comma separated list of label names.
+// Defaults to .producer,.app,.owner
+//
+#warp.labels.priority =
```

Comment:
the feature is not limited to WarpScript since it is also applies via the /fetch endpoint, so it is a general Warp config. Maybe we should move it back in another file.

Reply:
Are you sure?
`Configuration.WARP_LABELS_PRIORITY` is only used to build `FIND.DEFAULT_LABELS_PRIORITY` which is only used in `FIND.apply` and `FETCH.apply`, but not in `EgressFetchHandler`.
So it seems to me this is only a WarpScript configuration key.